packages/db/src/lib/execution.ts

```diff
@@ -61,33 +49,37 @@ export const getExecutionInfoByGranuleCumulusId = async ({
 };
 
 /**
- * Returns a list of executionArns sorted by most recent first, for an input
+ * Returns execution info sorted by most recent first for an input
  * Granule Cumulus ID.
  *
- * @param {Knex | Knex.Transaction} knexOrTransaction
+ * @param {Object} params
+ * @param {Knex | Knex.Transaction} params.knexOrTransaction
  *   Knex client for reading from RDS database
- * @param {number} granuleCumulusId - The primary ID for a Granule
- * @param {number} limit - limit to number of executions to query
- * @returns {Promise<ArnRecord[]>} - Array of arn objects with the most recent first.
+ * @param {Array<string>} params.executionColumns - Columns to return from executions table
+ * @param {number} params.granuleCumulusId - The primary ID for a Granule
+ * @param {number} [params.limit] - limit to number of executions to query
+ * @returns {Promise<Partial<PostgresExecutionRecord>[]>}
+ *   Array of arn objects with the most recent first.
  */
-export const getExecutionArnsByGranuleCumulusId = async (
+export const getExecutionInfoByGranuleCumulusId = async ({
```

Comment:
This should be noted in the CHANGELOG as a breaking change as it's a user exposed method.

Reply:
good point
https://github.com/nasa/cumulus/pull/3853/commits/d97a2aac1c77f8f47920355e9b0d7c883bd4b96d